parser.go

```diff
@@ -71,26 +72,24 @@ func NewErrorformatParserString(efms []string) (*ErrorformatParser, error) {
 	return NewErrorformatParser(efm), nil
 }
 
-func (p *ErrorformatParser) Parse(r io.Reader) ([]*CheckResult, error) {
+func (p *ErrorformatParser) Parse(r io.Reader) ([]*rdf.Diagnostic, error) {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported method ErrorformatParser.Parse should have comment or be unexported